Cartfile

```diff
@@ -1 +1 @@
-github "hyperwallet/hyperwallet-ios-sdk" "1.0.0-beta03"
+github "hyperwallet/hyperwallet-ios-sdk" "task/HW-54157-add-getField-in-transferMethod-and-user"
```

Comment:
this need to be fixed to development instead of task/HW-54157-add-getField-in-transferMethod-and-user